util/text.py

```diff
@@ -166,3 +167,25 @@ def range_less_than(previous_state, current_input):
     vals_sparse = gather_nd(labels, indices, shape)
     
     return tf.SparseTensor(tf.to_int64(indices), vals_sparse, tf.to_int64(label_shape))
+
+# Validate and normalize transcriptions. Returns a cleaned version of the label
+# or None if it's invalid.
+def validate_label(label):
```

Comment:
I'll land this first since it's 3AM PST and then Andre can just exclude this chunk from his PR.